docs/advanced/picking.md

```diff
@@ -73,12 +73,36 @@ calculatePickingColors(attribute) {
 
 ## Custom Shaders
 
-Deck.gl layers use luma.gl's [`Picking Module`](http://uber.github.io/luma.gl/#/documentation/api-reference/shader-module). If you are writing custom shaders for your layers, please refer to above luma.gl documentation on how to add picking logic to your shaders. Also make sure to create `Model` object with `picking` module. ()
+All core layers (including composite layers) support picking using luma.gl's `picking module`. If you are using custom shaders with any of the core layers or building custom layers with your own shaders following steps are needed to achieve `Picking`.
 
+### getShaders()
+
+`getShaders()` return value should include picking module in `modules` array.
```

Comment:
* Technically, `getShaders` is never called by the `Layer` lifecycle, it is just a convention in our layers. Could be worth mentioning.

Reply:
yeah, good point, actually what I wanted to document was to make sure `Model` object is created with `picking` module, updated accordingly.